scripts/standardise_validate.py

```diff
@@ -50,14 +50,19 @@ def main() -> None:
         # Validate the file
         file_check = FileCheck(file, scale, srs)
         if not file_check.validate():
-            get_log().info("non_visual_qa_errors", file=file_check.path, errors=file_check.errors)
+            # Format gdalinfo for logging
+            formatted_gdalinfo = file_check.get_gdalinfo()
+            if formatted_gdalinfo:
+                formatted_gdalinfo["coordinateSystem"]["wkt"] = format_wkt(formatted_gdalinfo["coordinateSystem"]["wkt"])
+                formatted_gdalinfo["stac"]["proj:wkt2"] = format_wkt(formatted_gdalinfo["stac"]["proj:wkt2"])
+            get_log().info("non_visual_qa_errors", file=file_check.path, errors=file_check.errors, gdalinfo=formatted_gdalinfo)
```

Comment:
Elastic Search does not like that a (some?) field name is empty:
`{"errors":{"bands":[{"band":1,"block":[512,512],"colorInterpretation":"Red","maximum":255,"mean":135.328,"metadata":{"":{ [...]`

Reply:
Not a good idea to log the full gdalinfo json anyway